src/aind_data_transfer/transcode/ome_zarr.py

```diff
@@ -295,16 +291,12 @@ def _compute_chunks(reader, target_size_mb):
 
 
 def _create_pyramid(data, n_lvls):
```

Comment:
I think this function only needs to be documented...

Reply:
done